src/dewey/curation.clj

```diff
@@ -57,12 +57,12 @@
   [irods es entity-type entity-id op]
   (log/trace "apply-or-remove <irods> <es>" entity-type entity-id "<op> called")
   (if-let [entity (entity/lookup-entity irods entity-type entity-id)]
-    (when (or (= type :data-object) (indexable? entity)) (op entity))
-    (indexing/remove-entity es entity-type entity-id)))
+    (when (or (= entity-type :data-object) (indexable? entity)) (op entity))
```

Comment:
I guess that before this was comparing with the [`type`](https://clojuredocs.org/clojure.core/type) builtin, heh. I think we should probably change this to:

```suggestion
    (when (indexable? entity) (op entity))
```

unrelated to your changes, but may as well clean it up now. I don't think the `or` really makes sense, and clearly before this it was only checking `indexable?`, so